docker/Dockerfile

```diff
@@ -216,6 +223,25 @@ RUN set -e; \
     esac; \
     truncate -s0 ~/.bash_history;
 
+RUN set -e; \
+    if ! which bazel >/dev/null ; then \
+        BAZEL_INSTALL_DIR='/usr/local/bin/'; \
+        BAZEL_VERSION='5.2.0'; \
+        BAZEL_INSTALL_PATH="$BAZEL_INSTALL_DIR/bazel-$BAZEL_VERSION"; \
+        BAZEL_ARCH=$(case "$TARGETARCH" in \
```

Comment:
先set -e;, subshell里不会把e带进去的

Reply:
There are tons of subshells in this file and none of them have `set -e`...

Reply:
1. Single-command may be handled differently for that.
2. Some commands are rock-solid or doesn't matter much.
3. set -e is a good trick for formatting, so that you can have the main code in next line...